v0_5/src/auction_proxy.rs

```diff
@@ -16,14 +16,17 @@ pub trait Auction {
     #[callback(auction_unstake_callback)]
     fn unStake(&self,
         #[callback_arg] node_ids: Vec<usize>,
-        #[var_args] bls_keys_signatures: VarArgs<BLSKey>);
+        #[var_args] bls_keys: VarArgs<BLSKey>);
 
     #[callback(auction_unbond_callback)]
     fn unBond(&self,
         #[callback_arg] node_ids: Vec<usize>,
         #[var_args] bls_keys_signatures: VarArgs<BLSKey>);
 
-    #[callback(auction_claim_callback)]
-    fn claim(&self,
-        #[callback_arg] node_ids: Vec<usize>);
+    fn claim(&self);
```

Comment:
no need for #callback(auction_claim:callback) ?

Reply:
or at least add someting like #callback[default_callback)]

Reply:
A default callback that does nothing is added automatically behind the scenes.
Will add some error handling in the default callback in the future, but for now it makes no difference it is specified with no implementation, or if it is not specified at all.